tests/wfc3/test_uvis_15single.py

```diff
@@ -3,7 +3,10 @@
 
 from ..helpers import BaseWFC3
 
-@pytest.mark.slow
+# This test is curretnly skipped as it contains 100 (!) files to 
```

Comment:
I was thinking more like `skip(reason='...')` but this works too. For reference: http://doc.pytest.org/en/latest/skipping.html#skipping-test-functions

```suggestion
# This test is currently skipped as it contains 100 (!) files to 
```

Reply:
Fixed typo and use pytest facilities when appropriate.